lib/iotagent-mqtt.js

```diff
@@ -26,6 +26,7 @@
 var iotAgentLib = require('iotagent-node-lib'),
     mqtt = require('mqtt'),
     dateFormat = require('dateformat'),
+    commandHandler = require('./commandHandlers'),
```

Comment:
It felt awkward to me when I read that the new file was called `commandHandlers.js` versus 'commandHandler.js`. Any reason to use the plural here?